docs/tables/scaleway_kubernetes_node.md

```diff
@@ -0,0 +1,66 @@
+# Table: scaleway_kubernetes_pool
+
+A Scaleway Kubernetes is a public cloud managed kubernetes available in Scaleway.
```

Comment:
@jplanckeel could we please update the table description here? It is exactly similar to the one we have used for the kubernetes_cluster table.

Reply:
yes is update ;)